modelzoo/DeepFM/train.py

```diff
@@ -210,122 +171,287 @@ def prediction(self):
-                        required=False)
-    parser.add_argument('--deep_dropout',
-                        help='Dropout regularization for deep model',
-                        type=float,
-                        default=0.0)
-    parser.add_argument("--optimizer",
-                        type=str,
-                        choices=["adam", "adagrad", "adamasync"],
-                        default="adam")
-    parser.add_argument('--learning_rate',
-                        help='Learning rate for model',
-                        type=float,
-                        default=0.001)
-    parser.add_argument('--save_steps',
-                        help='set the number of steps on saving checkpoints',
-                        type=int,
-                        default=0)
-    parser.add_argument('--keep_checkpoint_max',
-                        help='Maximum number of recent checkpoint to keep',
-                        type=int,
-                        default=1)
-    parser.add_argument('--bf16',
-                        help='enable DeepRec BF16 in deep model. Default FP32',
-                        action='store_true')
-    parser.add_argument('--no_eval',
-                        help='not evaluate trained model by eval dataset.',
-                        action='store_true')
-    parser.add_argument('--timeline',
-                        help='number of steps on saving timeline. Default 0',
-                        type=int,
-                        default=0)
-    parser.add_argument("--protocol",
-                        type=str,
-                        choices=["grpc", "grpc++", "star_server"],
-                        default="grpc")
-    parser.add_argument('--inter',
-                        help='set inter op parallelism threads.',
-                        type=int,
-                        default=0)
-    parser.add_argument('--intra',
-                        help='set inter op parallelism threads.',
-                        type=int,
-                        default=0)
-    parser.add_argument('--input_layer_partitioner',
-                        help='slice size of input layer partitioner. units MB',
-                        type=int,
-                        default=0)
-    parser.add_argument('--dense_layer_partitioner',
-                        help='slice size of dense layer partitioner. units KB',
-                        type=int,
-                        default=0)
-    return parser
+
+def build_feature_columns():
+    # Notes: Statistics of Kaggle's Criteo Dataset has been calculated in advance to save time.
+    mins_list = [
+        0.0, -3.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0
+    ]
+    range_list = [
+        1539.0, 22069.0, 65535.0, 561.0, 2655388.0, 233523.0, 26297.0, 5106.0,
+        24376.0, 9.0, 181.0, 1807.0, 6879.0
+    ]
+
+    def make_minmaxscaler(min, range):
+        def minmaxscaler(col):
+            return (col - min) / range
+
+        return minmaxscaler
+
```

Comment:
remove normalizer. Follow DeepFM code.

Reply:
it is removed

Reply:
mins_list, range_list and make_minmaxscaler() is still here